src/db.cpp

```diff
@@ -41,6 +41,8 @@ namespace mabain {
 // Current mabain version 1.1.0
 uint16_t version[4] = {1, 1, 0, 0};
 
+std::map<std::string, int> db_queue_size;
```

Comment:
It doesn't seem necessary that we need a map here to track the queue size. The queue size is already stored in the header. It should be validated when the header is mapped.
Besides, the C++ map cannot survive the process restart anyhow.